connectors/src/lib/models/microsoft.ts

```diff
@@ -83,6 +88,10 @@ MicrosoftRootModel.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
+    resourcePath: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
```

Comment:
Didn't we say we want to avoid using `resource` for this  ? can it be `nodePath` ?

Reply:
changed to `itemApiPath`